SwiftI18n/Classes/Views/SwiftUI/LocText.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SwiftUIView.swift
+//
+//
+//  Created by Antonijo Bezmalinovic on 14.04.2023..
+//
+
+import SwiftUI
+
+enum SwiftI18nKey: EnvironmentKey {
+    static var defaultValue: String = I18nManager.instance.language
+}
+
+public extension EnvironmentValues {
+    
+    var language: String {
+        get { self[SwiftI18nKey.self] }
+        set { self[SwiftI18nKey.self] = newValue }
+    }
+}
+
+public struct LanguageModifier: ViewModifier {
+    
+    @State private var language: String
+    
+    public init(language: String) {
+        self.language = language
+    }
```

Comment:
```suggestion
    @State private var language = I18nManager.instance.language
```
Since we will be changing language from inside, we don't need to send a default value in.